nhais/nhais-env.yaml

```diff
@@ -0,0 +1,5 @@
+#Common env variables
```

Comment:
To be consistent with other adaptors rename this file to nhais-env-example.yaml and .gitignore nhais/nhais-env.yaml. It doesn't contain anything sensitive now but will in the future.